src/content/developers/docs/smart-contracts/testing/index.md

```diff
@@ -46,4 +213,7 @@ incomplete: true
 
 ## Further reading {#further-reading}
 
-_Know of a community resource that helped you? Edit this page and add it!_
+- [An In-Depth Guide to Testing Ethereum Smart Contracts](https://iamdefinitelyahuman.medium.com/an-in-depth-guide-to-testing-ethereum-smart-contracts-2e41b2770297) - _Ben Hauser_
+- [How to Test Ethereum Smart Contracts](https://betterprogramming.pub/how-to-test-ethereum-smart-contracts-35abc8fa199d) - _Alex Roan_
+
+
```

Comment:
```suggestion
```

Also, just above in the "Related tutorials" we should list
```
- [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript (YouTube)](https://www.youtube.com/watch?v=gyMwXuJrbJQ)
- [Solidity, Blockchain, and Smart Contract Course (YouTube)](https://www.youtube.com/watch?v=M576WGiDBdQ)
```
Added the `(YouTube)` identifier there as a thought to just mark it as a video, could go either way on it. 

---

Also on that note, @PatrickAlphaC! If you see this, any input you'd be willing to give on this topic would be great, as I know you've educated on this topic before. 🙏 ❤️

Reply:
@wackerow added some notes below :D